core/arch/arm/kernel/boot.c

```diff
@@ -1508,6 +1508,14 @@ unsigned long __weak get_aslr_seed(void)
 		goto err;
 	}
 
+	/* 
+		Checks if seed is 8-byte aligned, and performs alignment if needed.
+		If removed while CFG_CORE_SANITIZE_UNDEFINED is set, it may cause a ubsan panic
+	*/
+	if (((uint64_t)seed & 7) != 0) {
+		seed = (uint64_t *)((unsigned long)seed + (8 - ((unsigned long)seed & 7)));
+	}
+	
 	return fdt64_to_cpu(*seed);
```

Comment:
You can't change the address of the seed. The correct fix is:
```suggestion
	return fdt64_to_cpu(fdt64_ld(seed));
```
Please make this a separate fix ("core: arm: use fdt64_ld() to read possibly unaligned kaslr-seed").